modules/katamari/src/main/ts/ephox/katamari/api/Singleton.ts

```diff
@@ -9,6 +9,13 @@ interface Singleton<T> {
   readonly set: (value: T) => void;
 }
 
+export interface Repeatable {
```

Comment:
@hamza0867 this PR is missing quite a number of things for this change... you've not added any Katamri tests nor a changelog entry. Please log a follow up PR to correct that.

Reply:
Also if making large changes after a review such as this it's normally a good idea to re-request reviews.